sys/include/crypto/3des.h

```diff
@@ -85,8 +85,8 @@ static const uint32_t bigbyte[24] = {
  * @param   keySize     the size of the key
 
  *
- * @return  Whether initialization was successful. The command may be
- *          unsuccessful if the key size is not valid.
+ * @return  CIPHER_INIT_SUCCESS if the initialization was successful.
+ *          The command may be unsuccessful if the key size is not valid.
```

Comment:
Done. I expanded your sentence a bit, to tell the user the reason for that problem.